traefik/values.yaml

```diff
@@ -72,6 +72,127 @@ volumes: []
 #   mountPath: "/config"
 #   type: configMap
 
+# Port Redirections
+# Added in 2.2, you can make permanent redirects via entrypoints.
+# https://docs.traefik.io/routing/entrypoints/#redirection
+redirections:
+  web:
```

Comment:
after some further testing and discussion, we should make that block easier. Just `entryPoint`based redirects (e.g. the `to`) won't work in all cases, as the traefik port is another one thatn the port of the outside service and thus, redirections wont work out.

Can you simplify this block to something, that only `activates` either a boolean to redirect to TLS or to redirect to a given port?

Reply:
Ive simplified to ignore adding priority/scheme through the chart as it makes sense that those options are probably almost ALWAYS default. Instead of "to" it shows "port" now and explicitly prefixes with a `:` in the templating to enforce.

Reply:
Thank you! One thing I know realize as you take the entryPoint name in the template.

We could move that part down to the actual port declaration because otherwise, we would have an implicit
relation between the two parts.

Then, we could fallback to the `to` value beeing the name of another port in that block. It could
look like this:

```
ports:
  web:
    port: 8000
    redirectTo: websecure
  websecure:
    port: 8443
    exposedPort: 443
```

With that beeing the case, we could do something like that in the template of the deployment (not tested ;-)) :

```yaml
{{ $toPort := index .Values.Ports $config.redirectTo }}
- "--entrypoints.{{ $name }}.http.redirections.entryPoint.to=:{{ $toPort.exposedPort }}"
- "--entrypoints.{{ $name }}.http.redirections.entryPoint.scheme=https"
```

Once that is resolved (and the comment from above), I think we're good to go :) Do you want me to do the change or can you do it?

Reply:
Makes sense. Ill make the changes this afternoon.